lib/spack/spack/test/link_paths.py

```diff
@@ -38,13 +39,17 @@ def check_link_paths(filename, paths):
     assert actual == expected
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
```

Comment:
Do these tests in fact fail on Windows? My understanding is that they just check parsing of output, which I thought would work on any system.

Reply:
Only due to path inconsistencies. I'll add a windows version of each path set.